src/Fl_Widget.cxx

```diff
@@ -18,6 +18,7 @@
 #include <FL/Fl_Widget.H>
 #include <FL/Fl_Group.H>
 #include <FL/Fl_Tooltip.H>
+#include <FL/Fl_Shared_Image.H>
```

Comment:
This should no longer be needed since the second commit doesn't use it.

Should be removed.